enhancements/network/vrf-cni.md

```diff
@@ -0,0 +1,123 @@
+---
+title: vrf-metaplugin-cni
+authors:
+  - "@fedepaol"
+reviewers:
+  - "@dougbtv"
+  - "@zshi-redhat"
+approvers:
+  - TBD
+creation-date: 2020-07-21
+last-updated: 2020-07-21
+status: implementable
+---
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
```

Comment:
Looks like this can be checked as the `Graduation Criteria` is defined in this enhancement doc.

Reply:
Right, thanks!